docs/sp-add-ins/sharepoint-admin-apis-authentication-and-authorization.md

```diff
@@ -0,0 +1,62 @@
+
+Users in a tenant can be granted administrator roles via [Microsoft admin center](/microsoft-365/admin/add-users/about-admin-roles) or [Microsoft Entra](/entra/fundamentals/users-assign-role-azure-portal). Not all administrator roles have access to all the admin APIs, and some administrator roles do not have access to any admin APIs at all. For example, users with [Global Reader](/entra/identity/role-based-access-control/permissions-reference#global-reader) role have access to read admin APIs, but have no access to admin APIs that make any updates.
+
+### Application access on behalf of a user
+
+Admin API operations [on behalf of a user](/graph/auth-v2-user) require applications to receive consent for SharePoint `AllSites.FullControl` application permission. This permission requires admin consent on the consuming tenant before any user from the tenant can consent to it.
+
+### Application access without a user
+
+Admin API operations [without a user](/graph/auth-v2-service) require applications to receive consent for Microsoft Graph [`Sites.FullControl.All`](/graph/permissions-reference#sitesfullcontrolall) application permission. This permission requires admin consent on the consuming tenant.
+
+> [!IMPORTANT]
+> When an application is to be used, using one with user-delegated application permissions is the recommended approach. This type of access enhances the security of your tenant and improves auditability of actions performed by the application. For all applications, an administrator on the consuming tenant must consent to your application's request for permission. Learn more [here](/entra/identity/enterprise-apps/grant-admin-consent?pivots=portal).
+
+> [!NOTE]
+> We are currently working on providing more granular, less-permissive scopes for applications to use based on what admin APIs the applications want to have access to. We'll share more information in due course when this is ready for adoption.
+
+## What's next
+
+Here are some actions you can take next to use applications for admin API calls:
+
+1. Configure your [application manifest](/entra/identity-platform/reference-app-manifest#requiredresourceaccess-attribute) to request the required permissions for Office 365 SharePoint Online (resourceAppId: `00000003-0000-0ff1-ce00-000000000000`).
+1. [Grant admin consent](/entra/identity/enterprise-apps/grant-admin-consent?pivots=portal) to your application on the target tenant.
+
+| Access type         | Permission name         | `resourceAccess` id                    | `resourceAccess` type |
+| :------------------ | :---------------------- | :------------------------------------- | :-------------------- |
+| On behalf of a user | `AllSites.FullControl`  | `56680e0d-d2a3-4ae1-80d8-3c4f2100e3d0` | Scope                 |
+| Without a user      | `Sites.FullControl.All` | `a82116e5-55eb-4c41-a434-62fe8a61c773` | Role                  |
+
+1. Use your application to [invoke admin APIs](./get-to-know-the-sharepoint-rest-service.md).
```

Comment:
The numbering restarts for this item in the preview and shows as 1 again. Try indenting the table from the previous point to be within its bullet point (so indent 3 spaces to the right). Then markdown may detect that this is another point in the original sequence.

Reply:
@dluces is correct... the table needs to be double-indented for the markdown rendering engine to continue the bullets

Reply:
Good catch! Fixed.